packages/html/src/pager/templates/pager-adaptive.tsx

```diff
@@ -0,0 +1,3 @@
+import Pager from "../pager.spec";
+
+export const PagerAdaptive = (props) => <Pager adaptive={true} info={false} refresh={false} itemsPerPage={false} {...props} />;
```

Comment:
Please use multiple lines, as it is easier for reading (like in here -https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/combobox/templates/combobox-adaptive.tsx#L3_